packages/@n8n/db/src/utils/get-final-test-result.ts

```diff
@@ -0,0 +1,30 @@
+import type { TestCaseExecution } from '../entities';
+import type { TestRunFinalResult } from '../entities/types-db';
+
+/**
+ * Returns the final result of the test run based on the test case executions.
+ * The final result is the most severe status among all test case executions' statuses.
+ */
+export function getTestRunFinalResult(testCaseExecutions: TestCaseExecution[]): TestRunFinalResult {
```

Comment:
this is identical to the original implementation in `packages/cli/src/evaluation.ee/test-runner/utils.ee.ts`, and I don't see it being used anywhere in `cli` anymore. 
Can we delete that copy, and move any tests for it into this package as well please 🙏🏽